readers/hdfs/src/test/java/com/criteo/hadoop/garmadon/hdfs/offset/HdfsOffsetComputerTest.java

```diff
@@ -22,11 +22,23 @@
 import static org.mockito.Mockito.when;
 
 public class HdfsOffsetComputerTest {
+    final HdfsOffsetComputer offsetComputer = new HdfsOffsetComputer(buildFileSystem(
```

Comment:
I dislike instance or class variables for tests. I would leave this where it was to ensure that there's no poisoning between tests